phpunit.xml.dist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@lcf since this was my first PR, as stated in the notes, I added this file to provide the ability to run phpunit tests from the root